Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -288,7 +288,7 @@ private void ScanForEntrance()
 
             var allowedDollarLoss = MaximumPorfolioRiskPercentPerPosition * Portfolio.TotalPortfolioValue;
 
-            var shares = (int) (allowedDollarLoss/expectedCaptureRange);
+            var shares = allowedDollarLoss/expectedCaptureRange;
```

Comment:
No problem. I think we can see in this case the potential for problems with changing the return type of CalculateOrderQuantity. It might work to add another method something like 

` CalculateOrderQuantity<decimal>(Symbol, MaximumLeverage)`